azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NextHop.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.network;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.HasParent;
+import com.microsoft.azure.management.resources.fluentcore.model.Executable;
+
+/**
+ * A client-side representation allowing user to get next hop for a packet from specific vm.
+ */
+@Fluent
+@Beta
```

Comment:
Regarding comments of getters in this file, see above mentioned note about keeping comment only for `return` is applicable.